git_machete/cmd.py

```diff
@@ -648,31 +649,128 @@ def annotate(b: str, words: List[str]) -> None:
+def create_github_pr(cli_ctxt: CommandLineContext, head: str, draft: bool) -> None:
+    global annotations, up_branch
+    from git_machete.github import GitHubPullRequest, add_assignees_to_pull_request, add_reviewers_to_pull_request, \
+        create_pull_request, derive_current_user_login, set_milestone_of_pull_request
+
+    base: Optional[str] = up_branch.get(head)
+    if not base:
+        raise MacheteException(f'Branch {head} does not have a parent branch (it is a root), base branch for the PR cannot be established')
+
     org: str
     repo: str
-    if len(org_name_for_github_remote) == 1:
-        org, repo = list(org_name_for_github_remote.values())[0]
-    elif len(org_name_for_github_remote) > 1:
-        if 'origin' in org_name_for_github_remote:
-            org, repo = org_name_for_github_remote['origin']
-        else:
-            raise MacheteException(f'Multiple non-origin remotes correspond to GitHub in this repository: '
-                                   f'{", ".join(org_name_for_github_remote.keys())}, aborting')
+    _, (org, repo) = derive_remote_and_github_org_and_repo(cli_ctxt)
     current_user: Optional[str] = derive_current_user_login()
-    debug(cli_ctxt, 'sync_annotations_to_github_prs()', 'Current GitHub user is ' + (current_user or '<none>'))
+    debug(cli_ctxt, f'create_github_pr({head})', f'organization is {org}, repository is {repo}')
+    debug(cli_ctxt, f'create_github_pr({head})', 'current GitHub user is ' + (current_user or '<none>'))
+
+    def slurp_file_or_empty(path: str) -> str:
+        try:
+            return open(path).read()
+        except IOError:
+            return ''
+
+    title: str = popen_git(cli_ctxt, "log", "-1", "--format=%s").strip()
+    description_path = get_git_subpath(cli_ctxt, 'info', 'description')
+    description: str = slurp_file_or_empty(description_path)
+
+    pr: GitHubPullRequest = create_pull_request(org, repo, head=head, base=base, title=title, description=description, draft=draft)
+    debug(cli_ctxt, f'create_github_pr({head})', f'created {pr}')
+
+    milestone_path: str = get_git_subpath(cli_ctxt, 'info', 'milestone')
+    milestone: str = slurp_file_or_empty(milestone_path).strip()
+    if milestone:
+        debug(cli_ctxt, f'create_github_pr({head})', f'setting milestone of PR #{pr.number} to {milestone}')
+        set_milestone_of_pull_request(org, repo, pr.number, milestone=milestone)
+
+    if current_user:
+        debug(cli_ctxt, f'create_github_pr({head})', f'adding {current_user} as assignee to PR #{pr.number}')
+        add_assignees_to_pull_request(org, repo, pr.number, [current_user])
+
+    reviewers_path = get_git_subpath(cli_ctxt, 'info', 'reviewers')
+    reviewers: List[str] = non_empty_lines(slurp_file_or_empty(reviewers_path))
+    if reviewers:
+        debug(cli_ctxt, f'create_github_pr({head})', f'adding {", ".join(reviewers)} as reviewers to PR #{pr.number}')
+        add_reviewers_to_pull_request(org, repo, pr.number, reviewers)
```

Comment:
Displaying progress won't be a bad idea here probably (esp. in case the operation is interrupted in-between, to see which steps already passed)